thread_safe_function_counting/node-addon-api/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "tsfn-counting",
+  "version": "0.0.0",
+  "description": "Thread-safe Function Example",
+  "main": "addon.js",
+  "private": true,
+  "gypfile": true,
+  "dependencies": {
+    "node-addon-api": "*",
+    "bindings": "*"
+  },
+  "engines": {
+    "node": ">= 10.6.0"
```

Comment:
This should probably be the version of Node where threadsafe functions where added or came out of experimental. I'm not sure what version that was. On my system it fails on v10.10.0 but works with v10.15.1.